docs/mtls.md

```
@@ -0,0 +1,138 @@
+  --tls \
+  --cert client-tls.crt \
+  --key client-tls.key \
+  --cacert ca.crt \
+  -h valkeycluster-mtls.default.svc.cluster.local \
+  -p 6379 \
+  PING
+```
+
+## Security considerations
+
+#### Never use `nopass: true` on cert-mapped users with either `authClientsUser: CN` or `authClientsUser: URI`
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)